config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -260,7 +260,7 @@ periodics:
   tags:
   - "perfDashPrefix: kubemark-5000Nodes"
   - "perfDashJobType: performance"
-  interval: 12h
+  cron: '1 12 1-31/2 * *' # Run every second day at 12:01 UTC starting on the 1st of each month
```

Comment:
Previously we had two runs per day, we this we'll have only one.
I know that cron is not flexible enough to express what we'd want (e.g. keep running twice a day, every 3rd run run scheduler suite instead of regular test). Ideally, we'd be able to specify multiple cron expressions, but prow doesn't support it. But, can we at least do something like:

1. Run the regular test twice a day every second (odd) day, given that it takes ~14h, let's run it 00:01 and 16:01
2. Run the scheduler test once a day every second (even) day, to give the previous regular tests 14h +2h buffer to finish let's start it at 08:01

WDYT?

Reply:
sg :+1: but instead of 08:01 UTC for the scheduler suite job, I opted for a more conservative 10:01 to give it a bit more time. This shouldn't be an issue because the scheduler suite job is expected to finish in 6-8 hours, which is well before the next kubemark-5000Nodes job.